Automattic/pocket-casts-ios
# Search Improvements: design tweaks

Fixes some of the issues mentioned here: https://github.com/Automattic/pocket-casts-ios/issues/709#issuecomment-1495723140

* Updated icon
* Updated shadows
* Updated leading/trailing space
* Increased podcast covers

https://user-images.githubusercontent.com/7040243/229880523-9f9b26fa-86e8-49d1-a6db-06a2f99fb4fc.mp4

## To test

1. Run the app
2. Search for podcasts, folders, or episodes
3. ✅ Check the big covers and the small ones
4. Tap on "Show all" in both podcasts and episodes
5. ✅ Check that the covers appear just fine with a lighter shadow

## Checklist
